src/main/java/org/killbill/billing/entitlement/api/EntitlementApi.java

```diff
@@ -59,6 +59,22 @@ public Entitlement createBaseEntitlement(UUID accountId, PlanPhaseSpecifier spec
             throws EntitlementApiException;
 
     /**
+     * Create a new base entitlement and addOn entitlements for that account.
+     * <p/>
```

Comment:
This will unfortunately break on JDK 1.8. You need to use `<p>` instead of `<p/>`.